<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskEditDialog</class>
 <widget class="QDialog" name="TaskEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="title_label">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="title_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="desc_label">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="desc_plainTextEdit"/>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="hour_spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> hours</string>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="min_spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> mins</string>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sec_spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> secs</string>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="useTimeLimit_checkBox">
         <property name="text">
          <string>set time limit to the task</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="completed_checkBox">
         <property name="text">
          <string>task is completed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="timeList_tab">
      <attribute name="title">
       <string>Time intervals</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeWidget" name="taskTime_treeWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Start time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>End time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Length</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QToolButton" name="add_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="edit_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="remove_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeAll_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="add_action">
   <property name="text">
    <string>&amp;Add...</string>
   </property>
   <property name="toolTip">
    <string>Add</string>
   </property>
  </action>
  <action name="remove_action">
   <property name="text">
    <string>&amp;Remove...</string>
   </property>
   <property name="toolTip">
    <string>Remove</string>
   </property>
  </action>
  <action name="edit_action">
   <property name="text">
    <string>&amp;Edit...</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="removeAll_action">
   <property name="text">
    <string>Remove &amp;all...</string>
   </property>
   <property name="toolTip">
    <string>Remove all</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>title_lineEdit</tabstop>
  <tabstop>useTimeLimit_checkBox</tabstop>
  <tabstop>hour_spinBox</tabstop>
  <tabstop>min_spinBox</tabstop>
  <tabstop>sec_spinBox</tabstop>
  <tabstop>desc_plainTextEdit</tabstop>
  <tabstop>completed_checkBox</tabstop>
  <tabstop>taskTime_treeWidget</tabstop>
  <tabstop>add_toolButton</tabstop>
  <tabstop>edit_toolButton</tabstop>
  <tabstop>remove_toolButton</tabstop>
  <tabstop>removeAll_toolButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTimeLimit_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hour_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTimeLimit_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>min_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTimeLimit_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>sec_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
